notebooks/mock-notebooks/login/User Login Flow.ipynb

```diff
@@ -0,0 +1,522 @@
+{
```

Comment:
<p>This email gets pulled from the data / domain owner user information set in the UI? (Shubham suggested an explicit field called Support Email.) Unless the domain doesn't exist. In which case.... the network support email?</p><p> </p><p>Also, if the user is getting the URL wrong (as we discussed, if they're getting a 404 error) we want to nudge them to check the URL. So "seems to be offline at the current moment. Please check your URL and ensure that it's correct."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Same as above I'm down to pivot in that direction. Although might want to include text that gets at the idea that they may have to wait for the node to be online. As in if they do have the correct URL and the domain is simply not online at the moment they should be aware that it may not be a user error but just that the node is not online and they need to check back in later.</p>